shell/src/main/java/alluxio/cli/fs/command/LoadCommand.java

```diff
@@ -138,4 +171,33 @@ public String getDescription() {
   public void validateArgs(CommandLine cl) throws InvalidArgumentException {
     CommandUtils.checkNumOfArgsNoLessThan(this, cl, 1);
   }
+
+  private void cacheBlock(long blockId, WorkerNetAddress dataSource, boolean local,
+      URIStatus status, Protocol.OpenUfsBlockOptions options) {
+    BlockInfo info = status.getBlockInfo(blockId);
+    long blockLength = info.getLength();
+    CacheRequest request = CacheRequest.newBuilder()
+                                      .setBlockId(blockId)
+                                      .setLength(blockLength)
+                                      .setOpenUfsBlockOptions(options)
+                                      .setSourceHost(dataSource.getHost())
+                                      .setSourcePort(dataSource.getDataPort()).build();
+    // TODO(jianjian) support FUSE processLocalWorker?
```

Comment:
why is FUSE processLocalWorker related here?

Reply:
remove comment